### DISCUSSION TITLE
Bookmarked POI gets changed or deleted in the map, what happens then to my bookmark?

### DISCUSSION THREAD

**Discussion by ivanbistrovic** [organicmaps/organicmaps#5632] [Q&A] [answered]
I had some bookmarks that might have dissapeared and I don't know the reason, altough I might be wrong, but just for prevention reasons I am posting this question.
As an example, a certain restaurant relocated to another address and on bookmarked address is no registered POI anymore, or on the same address another business opened.
So the question that came to me is what happens when a bookmarked location (eg. POI) gets deleted or changed in the map?
Sooner or later things change on the map, so I don't want to end up with no bookmarks on my map. 
As an example, when that happenes in Google Maps, then from the bookmarked location only coordinates of that point on the map is left.
In that case I can at least have a location which can help me identify and keep track of what was there and why I bookmarked it in the first place.

Cheers

**Comment by biodranik** [comment, marked as answer]
Bookmarks do not disappear in OM. They store their initial information. 

Bookmarks can disappear only if you delete them, or in very rare case of storing invalid XML symbols in bookmark names or descriptions (the whole list disappears in this case). Or in even more rare case of write file corruption.
